lib/topology_docker_openswitch/openswitch.py

```diff
@@ -221,6 +280,9 @@ def __init__(
             **kwargs
         )
 
+        # Save location of the shared dir in host
+        self.shared_dir = shared_dir
+
```

Comment:
Where do you recommend saving this? We need the logs per container and in /tmp/topology_{}_{} seemed like a safe place to do this. And the logs are produced inside the docker container so we need a shared_dir to get the log file.